src/mapml-viewer.js

```diff
@@ -525,6 +547,39 @@ export class MapViewer extends HTMLElement {
               {target: this}}));
       }
     });
+    var locateControl = this._geolocationButton._container;
+    var observer = new MutationObserver(function(mutations) {
+      // Check the current state of the control
+      if (locateControl.classList.contains('active') && locateControl.classList.contains('following')) {
+        locateControl.firstChild.title = M.options.locale.btnLocTrackOn;
+      } else if (locateControl.classList.contains('active')) {
+        locateControl.firstChild.title = M.options.locale.btnLocTrackLastKnown;
+      } else {
+        locateControl.firstChild.title = M.options.locale.btnLocTrackOff;
+      }
+    });
+    // Configure the observer to watch for changes to the class name
+    var observerConfig = { attributes: true, attributeFilter: ['class'] };
+    observer.observe(locateControl, observerConfig);
+
+    this._map.on('locationfound',
+      function (e) {
+        this.dispatchEvent(new CustomEvent('maplocationfound', {detail:
```

Comment:
The events are working pretty nicely!
I dont think this:

```js
this.addEventListener('maplocationfound', function(e) {
  //'Location found:', e.detail.latlng, 'Radius:', e.detail.radius
});
```
and this:
```js
this.addEventListener('locationerror', function(e) {
  //error:e.detail.error
});
```
is needed, as this is something the users will add.